diff --git a/tools/jarjar/jarjar.bzl b/tools/jarjar/jarjar.bzl
index 72eb7c6..dd55e11 100644
--- tools/jarjar/jarjar.bzl
+++ tools/jarjar/jarjar.bzl
@@ -76,7 +76,7 @@ jarjar_library = rule(
             allow_files = [".jar"],
         ),
         "_jarjar": attr.label(
-            default = Label("//tools/jarjar"),
+            default = Label("@bazel_tools//third_party/jarjar:jarjar_bin"),
             executable = True,
             cfg = "host",
         ),
diff --git a/tools/javadoc/javadoc.bzl b/tools/javadoc/javadoc.bzl
index 67f5aac..1f1b984 100644
--- tools/javadoc/javadoc.bzl
+++ tools/javadoc/javadoc.bzl
@@ -65,6 +65,8 @@ def _javadoc_library(ctx):
     jar_command = "%s/bin/jar cf %s -C tmp ." % (java_home, ctx.outputs.jar.path)
 
     srcs = depset(transitive = [src.files for src in ctx.attr.srcs]).to_list()
+    for src in srcs:
+      javadoc_command.append(src.path)
     ctx.actions.run_shell(
         inputs = srcs + classpath + ctx.files._jdk,
         command = "%s && %s" % (" ".join(javadoc_command), jar_command
