--- tools/jarjar/jarjar.bzl
+++ tools/jarjar/jarjar.bzl
@@ -79,7 +79,7 @@
 # Additional attributes only used in opensource builds
 _jarjar_library_attrs.update({
     "_jarjar": attr.label(
-        default = Label("//tools/jarjar"),
+        default = Label("@bazel_tools//third_party/jarjar:jarjar_bin"),
         executable = True,
         cfg = "host",
     ),

--- tools/maven/pom_file.bzl
+++ tools/maven/pom_file.bzl
@@ -41,7 +41,7 @@
 
 def _collect_maven_info_impl(_target, ctx):
     tags = getattr(ctx.rule.attr, "tags", [])
-    deps = getattr(ctx.rule.attr, "deps", [])
+    deps = getattr(ctx.rule.attr, "deps", []) + getattr(ctx.rule.attr, "runtime_deps", [])
     exports = getattr(ctx.rule.attr, "exports", [])
 
     maven_artifacts = []
@@ -60,6 +60,7 @@
     attr_aspects = [
         "deps",
         "exports",
+        "runtime_deps",
     ],
     doc = """
     Collects the Maven information for targets, their dependencies, and their transitive exports.
